### PR TITLE
Remove unused dependency 'abort-controller'

### PR DESCRIPTION
**Pull request by ymqy**
## Summary
This PR removes the unused dependency 'abort-controller' from the project. it helps to keep the project clean and maintainable.

## How did you test this change?
ci green

### CODE REVIEW

**Comment by ymqy** [comment]
CI pipeline failed at the checkout code step. Please review the logs and provide necessary updates to resolve the issue.

image

File: package.json [hunk 1]
```diff
@@ -36,7 +36,6 @@
     "@babel/preset-flow": "^7.10.4",
     "@babel/preset-react": "^7.10.4",
     "@babel/traverse": "^7.11.0",
-    "abort-controller": "^3.0.0",
```
Comment (eps1lon) [inline package.json:39]: Added in https://github.com/facebook/react/pull/21027 but it doesn't look like it was even used in that PR. Maybe something used during development?

Comment (ymqy) [reply package.json:39]: @eps1lon  “Adds a lightweight fallback shim for environments that don't have an AbortController (e.g. tests running in Node < 15).” from https://github.com/facebook/react/pull/24285

Comment (sebmarkbage) [reply package.json:39]: Hm. Interestingly that fix is only on the client. It didn’t necessarily fix the server. I guess we just don’t run tests in earlier node environments and we don’t really support them anyway.

Comment (ymqy) [reply package.json:39]: It should be safe to remove this dependency

Comment (kassens) [reply package.json:39]: Given that nothing requires this dependency at the moment, it seems like we can remove it at least until we do require it again. Seems like that might be required for some old node versions, but it seems like they'd need to be tested more extensively anyway.

If someone want's to use abort-controller again, it'd easy to add back the dependency. I'll merge this.

**Pull request merged**